server/index.ts

```diff
@@ -7,17 +7,18 @@ import { getTodayEventsJob } from './jobs';
 
 const app: Express = express();
 const PORT = process.env.PORT || 3000;
+const HOST = process.env.HOST || 'localhost:3000';
 
-const BACKEND_URL =
+const FRONTEND_URL =
   process.env.NODE_ENV === 'production'
     ? process.env.URL
-    : 'http://localhost:3000';
+    : 'http://localhost:5173';
```

Comment:
another way of doing this is:

FRONTEND_URL = process.env.URL ?? 'http://localhost:5173'

meaning if no URL has been defined it's localhost:5173 
this way we can change the URL regardless of whether it's production or not.